Code/GraphMol/ROMol.h

```diff
@@ -322,6 +323,69 @@ class RDKIT_GRAPHMOL_EXPORT ROMol : public RDProps {
   //! construct a molecule from a pickle string
   ROMol(const std::string &binStr, unsigned int propertyFlags);
 
+  ROMol(ROMol &&o) noexcept
+      : RDProps(std::move(o)),
+        d_graph(std::move(o.d_graph)),
```

Comment:
I'm not sure we're zeroing out any pointers here on the original molecule.  Should we?

Reply:
swapping to using exchange does so